migrations/versions/201511021701_134a1c372738_add_currency_to_registration.py

```diff
@@ -0,0 +1,43 @@
+"""Add currency to registration
+
+Revision ID: 134a1c372738
+Revises: 38641fb3afac
+Create Date: 2015-11-02 17:01:41.117908
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '134a1c372738'
+down_revision = '38641fb3afac'
```

Comment:
Should be 98f411f40bb (and I'd also update the date in the filename to keep a proper order).